pkg/scalers/metrics_api_scaler.go

```diff
@@ -0,0 +1,144 @@
+package scalers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	kedautil "github.com/kedacore/keda/pkg/util"
+	"io/ioutil"
+	"k8s.io/api/autoscaling/v2beta2"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+	"net/http"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"strconv"
+	"strings"
+)
+
+type httpScaler struct {
+	metadata *httpScalerMetadata
+}
+
+type httpScalerMetadata struct {
+	targetValue int
+	apiURL      string
+	metricName  string
+}
+
+type metric struct {
+	Name  string  `json:"name"`
+	Value float64 `json:"value"`
+}
+
+var httpLog = logf.Log.WithName("http_scaler")
+
+// NewHTTPScaler creates a new HTTP scaler
+func NewHTTPScaler(resolvedEnv, metadata, authParams map[string]string) (Scaler, error) {
```

Comment:
Should we change it to?
```suggestion
func NewMetricAPIScaler(resolvedEnv, metadata, authParams map[string]string) (Scaler, error) {
```

Reply:
+1 from me